### PR TITLE
Abdul/spi display

### PR DESCRIPTION
**Pull request by v-abdrahman**
Created a folder for SPI Display app & added all the project files for windows 10 Iot Enterprise.

Following changes are done:

1) file modified : SPIDisplay.csproj
    TargetPlatformVersion from 10.0.17763.0 to 10.0.19041.0 and TargetPlatformMinVersion from 10.0.10240.0 to 10.0.15063.0.

2) file modified : README.md
    Adding windows 10 iot Enterprise in products.
    Updated the Windows Platform from IoT Core to IoT Enterprise
    Added the BIOS setting for UPBOARD Pin configuration
    Added the app *packages installing steps

### CODE REVIEW

File: SPIDisplay/CS/README.md [hunk 1]
```diff
@@ -0,0 +1,454 @@
+---
```
Comment (rsameser) [inline SPIDisplay/CS/README.md:1]: Can you make sure all images in this ReadMe populate?

**Review by rsameser — CHANGES_REQUESTED** [review]
Please make sure all images populate in CS/ReadMe file

File: SPIDisplay/CS/README.md (hunk 1 above)
Comment (v-abdrahman) [reply SPIDisplay/CS/README.md:1]: Updated the image paths and images are coming up in readme.

**Comment by v-abdrahman** [comment]
Updated the image paths and images are coming up in readme.

**Pull request merged**